x-pack/legacy/plugins/actions/server/builtin_action_types/es_index.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema, TypeOf, Type } from '@kbn/config-schema';
+
+import { ActionType, ActionTypeExecutorOptions, ActionTypeExecutorResult } from '../types';
+
+function nullableType<V>(type: Type<V>) {
```

Comment:
should move this to a shared location rather than copying this everywhere.

Reply:
hoping we won't need it at all, eventually: https://github.com/elastic/kibana/pull/41728

But yeah, for now, will find a place to put it to be shared.

Reply:
moved to a new stand-alone module